code/modules/mob/living/simple_animal/hostile/bees.dm

```diff
@@ -137,21 +134,6 @@
 /mob/living/simple_animal/hostile/poison/bees/worker
 	//Blank type define in case we need to give them special stuff later, plus organization (currently they are same as base type bee)
 
-
-/mob/living/simple_animal/hostile/poison/bees/worker/Destroy()
```

Comment:
always have reference clearing handled on the highest type possible, even if it's not *directly* used on that highest type.